src/main/kotlin/test/project/findValue/database/dto/ValueInFiles.kt

```diff
@@ -0,0 +1,15 @@
+package test.project.findValue.database.dto
+
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+
+import javax.persistence.Id
+
+@Entity
+class ValueInFiles{
+    @Id
+    @GeneratedValue
+    var id: Long? = null
```

Comment:
This field is uncessesary. This is many-to-many binding. 
TableA (a_id number, ....) Table_B(b_id number, ...) many-to-many TableC(a_id, b_id)